core/src/test/java/kafka/test/ClusterConfig.java

```diff
@@ -53,7 +55,7 @@ public class ClusterConfig {
 
     ClusterConfig(Type type, int brokers, int controllers, String name, boolean autoStart,
                   SecurityProtocol securityProtocol, String listenerName, File trustStoreFile,
-                  String ibp) {
+                  String ibp, MetadataVersion metadataVersion) {
```

Comment:
It seems like we should parse the IBP string and deduce `MetadataVersion`, right?

Reply:
Hm, actually I think I'll remove the ibp part from the annotation and just use MetadataVersion